recipes/pyreadstat/meta.yaml

```diff
@@ -0,0 +1,105 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "pyreadstat" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: 8e48f4ad426dce82ee0036f5874223fd7688709cc4a7a337fd9896cefca73596
```

Comment:
You need to use these commented out lines here and build from source.

We cannot use the wheels provided on PyPI because they are not guaranteed to be binary compatible with the other  packages (like Python itself) in conda-forge.

Reply:
Build from source will fail for windows since the package does not compile with MSVC. 

So I guess that makes the package incompatible with conda forge?

Reply:
We also have the cygwin / m2 compilers available, I believe, if what you mean by `compile with MSVC` is that it requires gcc on windows.

Reply:
correct, theoretically cygwin should be able to compile it (right now I am doing with msys2-mingw64), however in order to do it I have to tweak quiet a few things in distutils, since it seems that by default you cannot build python extensions with mingw but only with microsoft visual studio? I guess such tweaks would not be available on appveyor? Is there any example out there of somebody building a cython extension this way (cygwin on appveyor)?

Reply:
Those patches are already in the conda-forge python package.

Reply:
I see, I will give it a go trying to build from source.
How do I set the compiler to cygwin/gcc?

Reply:
They are definitely available on appveyor.  You can check out https://github.com/conda-forge/r-ics-feedstock/blob/master/recipe/meta.yaml for an example of what an R recipe does.  I am not sure about a Cython example specifically.  Maybe try asking on Gitter: https://gitter.im/conda-forge/conda-forge.github.io

Reply:
yeah, but R is different, by default it always compiles with mingw on Windows, while python is configured to compile with microsoft visual studio, and getting it to compile with mingw requires making manual changes on distutils (but @isuruf implied that maybe that is already done?). 
Right now trying to compile with - {{ compiler('c') }} and cxx both still use microsoft visual studio. 

I ask on gitter as suggested.

Reply:
here just for documentation: suggestion from @mingwandroid: look at this example recipe https://github.com/conda-forge/pywinpty-feedstock/tree/master/recipe
from @isuruf: Yes, there's {{ compiler("m2w64_c") }} and m2w64_fortran and m2w64_cxx

Making the current recipe as pywinpty makes it to compile in appveyor!